src/vs/workbench/contrib/terminal/browser/links/terminalShellIntegrationLinkDetector.ts

```diff
@@ -26,18 +40,19 @@ export class TerminalShellIntegrationLinkDetector implements ITerminalLinkDetect
 	}
 
 	detect(lines: IBufferLine[], startLine: number, endLine: number): ITerminalSimpleLink[] {
-		if (this._matches(lines)) {
+		const match = this._matches(lines);
+		if (match) {
 			return [{
-				text: linkText,
+				text: match,
 				type: TerminalBuiltinLinkType.Url,
-				label: localize('learn', 'Learn about shell integration'),
+				label: match === LinkType.Activated ? localize('learn', 'Learn about shell integration') : localize('read more', 'Read more about this warning'),
 				uri: URI.from({
 					scheme: Schemas.https,
-					path: 'aka.ms/vscode-shell-integration'
+					path: match === LinkType.Activated ? 'aka.ms/vscode-shell-integration' : 'aka.ms/vscode-shell-integration#_complex-bash-promptcommand'
```

Comment:
@Tyriar how do I make a custom redirect like this?

Reply:
Sent you a dm